packages/web-components/fast-components/src/picker/picker-menu.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css, ElementStyles } from "@microsoft/fast-element";
+import {
+    ElementDefinitionContext,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+    FoundationElementDefinition,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import {
+    accentFillActive,
+    bodyFont,
+    controlCornerRadius,
+    designUnit,
+    focusStrokeOuter,
+    focusStrokeWidth,
+    foregroundOnAccentActive,
+    neutralForegroundRest,
+    neutralLayer3,
+    neutralLayerFloating,
+    strokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+} from "../design-tokens";
+import { elevation, heightNumber } from "../styles/index";
+
+export const pickerMenuStyles: (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => ElementStyles = (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) =>
+    css`
+        :host {
+            background: ${neutralLayerFloating};
+            --elevation: 11;
+            z-index: 1000;
+            display: flex;
+            width: 100%;
+            max-height: 100%;
+            min-height: 58px;
```

Comment:
Should this be a factor of the line-height?

Reply:
Yep